packages/p2p/src/stores/order-store.js

```diff
@@ -83,6 +83,21 @@ export default class OrderStore {
         this.getWebsiteStatus(setShouldShowCancelModal);
     }
 
+    @action.bound
+    getP2POrderList() {
```

Comment:
This api is already subscribed to in general store, but it doesn't update the current list. had to force update the order list as when the user rates an order, it didn't update the list to display the current recommendation.

Reply:
I tried to subscribe but it's still the same issue here, and when calling subscribe again in order store, it returns you are already subscribed